distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -281,10 +281,24 @@ public void testReadDerivedStorage() throws IOException {
   @Test
   public void testMetacardTransformersFromRest() {
```

Comment:
✏️ Change the name to something more specific, like `testGeoJsonMetacardTransformerFromRest`